stream-chat-android-ui-components/src/main/kotlin/io/getstream/chat/android/ui/message/list/MessageListView.kt

```diff
@@ -700,6 +712,15 @@ public class MessageListView : ConstraintLayout {
         }
     }
 
+    private fun updateMessageOptionsConfiguration(
+        reducer: MessageOptionsView.Configuration.() -> MessageOptionsView.Configuration,
+    ) {
+        check(::messageOptionsConfiguration.isInitialized.not()) {
```

Comment:
🤔 Why are we forcing to update that before configuration is initialized? I think it will be working fine even if configuration is already initialized

Reply:
Sorry, silly mistake. Noticed just after I created the PR. Will test again and submit.